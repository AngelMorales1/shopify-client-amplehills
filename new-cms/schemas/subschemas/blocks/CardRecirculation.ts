import image from './../image';
import color from './../color';

export default {
  type: 'object',
  name: 'cardRecirculation',
  title: 'Card Recirculation',
  fieldsets: [
    {
      name: 'card1',
      title: 'Card 1',
      options: { collapsible: true, collapsed: true }
    },
    {
      name: 'card2',
      title: 'Card 2',
      options: { collapsible: true, collapsed: true }
    }
  ],
  fields: [
    {
      name: 'title',
      title: 'Title',
      type: 'string'
    },
    {
      name: 'text1',
      title: 'Text 1',
      fieldset: 'card1',
      type: 'string'
    },
    image({
      name: 'image1',
      title: 'Image 1',
      fieldset: 'card1'
    }),
    color({
      name: 'color1',
      title: 'Color 1',
      fieldset: 'card1'
    }),
    {
      name: 'link1',
      title: 'Link 1',
      fieldset: 'card1',
      type: 'url',
      validation: Rule => Rule.uri({ allowRelative: true })
    },
    {
      name: 'text2',
      title: 'Text 2',
      fieldset: 'card2',
      type: 'string'
    },
    image({
      name: 'image2',
      title: 'Image 2',
      fieldset: 'card2'
    }),
    color({
      name: 'color2',
      title: 'Color 2',
      fieldset: 'card2'
    }),
    {
      name: 'link2',
      title: 'Link 2',
      fieldset: 'card2',
      type: 'url',
      validation: Rule => Rule.uri({ allowRelative: true })
    },
  ]
};
