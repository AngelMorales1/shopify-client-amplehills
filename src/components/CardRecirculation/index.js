import React from 'react';
import PropTypes from 'prop-types';
import get from 'utils/get';
import contentfulImgUtil from 'utils/contentfulImgUtil';
import cx from 'classnames';
import imageModel from 'models/imageModel';

import styles from './CardRecirculation.scss';
import { Button, Image } from 'components/base';

const CardRecirculation = ({ block }) => {
  const fields = get(block, 'fields', {});
  const card1HasData = Object.keys(fields).find(field =>
    field.includes('card1')
  );
  const card1Color = get(fields, 'card1Color', 'dark-blue');
  const card2HasData = Object.keys(fields).find(field =>
    field.includes('card2')
  );
  const card2Color = get(fields, 'card2Color', 'dark-blue');

  return (
    <div
      className={cx(
        styles['CardRecirculation'],
        'w100 p3 mt4 flex content-width mx-auto'
      )}
    >
      {card1HasData ? (
        <div className="col-12 md-col-6 m1">
          <Button
            to={get(fields, 'card1Link', '')}
            className="text-white w100"
            variant="style-none"
          >
            <div
              className={cx(
                styles['CardRecirculation__card'],
                styles[`CardRecirculation__card--${card1Color}`],
                'flex flex-column items-center justify-between p3 w100'
              )}
            >
              <Image
                className={cx(styles['CardRecirculation__card-image'], 'mt2')}
                src={contentfulImgUtil(
                  get(fields, 'card1Image.fields.file.url', ''),
                  '300',
                  'png'
                )}
              />
              <p className="py2">{get(fields, 'card1Text', '')}</p>
            </div>
          </Button>
        </div>
      ) : null}
      {card2HasData ? (
        <div className="col-12 md-col-6 m1">
          <Button
            to={get(fields, 'card2Link', '')}
            className="text-white w100"
            variant="style-none"
          >
            <div
              className={cx(
                styles['CardRecirculation__card'],
                styles[`CardRecirculation__card--${card2Color}`],
                'flex flex-column items-center justify-between p3 w100'
              )}
            >
              <Image
                className={cx(styles['CardRecirculation__card-image'], 'mt2')}
                src={contentfulImgUtil(
                  get(fields, 'card2Image.fields.file.url', ''),
                  '300',
                  'png'
                )}
              />
              <p className="py2">{get(fields, 'card2Text', '')}</p>
            </div>
          </Button>
        </div>
      ) : null}
    </div>
  );
};

CardRecirculation.propTypes = {
  fields: PropTypes.shape({
    card1Color: PropTypes.string,
    card1Image: imageModel.propTypes,
    card1Link: PropTypes.string,
    card1Text: PropTypes.string,
    card2Color: PropTypes.string,
    card2Image: imageModel.propTypes,
    card2Link: PropTypes.string,
    card2Text: PropTypes.string,
    title: PropTypes.string
  })
};

CardRecirculation.defaultProps = {
  fields: {
    card1Color: '',
    card1Image: imageModel.default,
    card1Link: '',
    card1Text: '',
    card2Color: '',
    card2Image: imageModel.default,
    card2Link: '',
    card2Text: '',
    title: ''
  }
};

export default CardRecirculation;
