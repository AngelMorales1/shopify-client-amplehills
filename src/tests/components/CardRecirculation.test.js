import React from 'react';
import { shallow } from 'enzyme';

import CardRecirculation from 'components/CardRecirculation';

it('renders without data', () => {
  const component = shallow(<CardRecirculation />);

  expect(component).toMatchSnapshot();
});

it('renders with data', () => {
  const component = shallow(
    <CardRecirculation
      block={{
        fields: {
          card1Color: 'foo',
          card1Image: {
            fields: {
              file: {
                url: '/foo'
              }
            }
          },
          card1Link: 'foo',
          card1Text: 'foo',
          card2Color: 'foo',
          card2Image: {
            fields: {
              file: {
                url: '/foo'
              }
            }
          },
          card2Link: 'foo',
          card2Text: 'foo'
        }
      }}
    />
  );

  expect(component).toMatchSnapshot();
});
