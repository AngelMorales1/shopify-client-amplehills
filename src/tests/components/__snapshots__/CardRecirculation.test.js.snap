// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with data 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CardRecirculation
    block={
        Object {
            "fields": Object {
              "card1Color": "foo",
              "card1Image": Object {
                "fields": Object {
                  "file": Object {
                    "url": "/foo",
                  },
                },
              },
              "card1Link": "foo",
              "card1Text": "foo",
              "card2Color": "foo",
              "card2Image": Object {
                "fields": Object {
                  "file": Object {
                    "url": "/foo",
                  },
                },
              },
              "card2Link": "foo",
              "card2Text": "foo",
            },
          }
    }
    fields={
        Object {
            "card1Color": "",
            "card1Image": Object {
              "fields": Object {
                "file": Object {
                  "url": "",
                },
                "title": "",
              },
              "sys": Object {
                "id": "",
              },
            },
            "card1Link": "",
            "card1Text": "",
            "card2Color": "",
            "card2Image": Object {
              "fields": Object {
                "file": Object {
                  "url": "",
                },
                "title": "",
              },
              "sys": Object {
                "id": "",
              },
            },
            "card2Link": "",
            "card2Text": "",
            "title": "",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="col-12 md-col-6 m1"
>
          <Button
                    className="text-white w100"
                    color="white-denim"
                    disabled={false}
                    fullWidth={false}
                    hover="shadow"
                    label=""
                    onClick={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    shadow={false}
                    to="foo"
                    type="button"
                    variant="style-none"
          >
                    <div
                              className="flex flex-column items-center justify-between p3 w100"
                    >
                              <Image
                                        alt=""
                                        bg={false}
                                        className="mt2"
                                        onImgLoad={[Function]}
                                        src="/foo?w=300&fm=png&q=90"
                                        style={Object {}}
                              />
                              <p
                                        className="py2"
                              >
                                        foo
                              </p>
                    </div>
          </Button>
</div>,
        <div
          className="col-12 md-col-6 m1"
>
          <Button
                    className="text-white w100"
                    color="white-denim"
                    disabled={false}
                    fullWidth={false}
                    hover="shadow"
                    label=""
                    onClick={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    shadow={false}
                    to="foo"
                    type="button"
                    variant="style-none"
          >
                    <div
                              className="flex flex-column items-center justify-between p3 w100"
                    >
                              <Image
                                        alt=""
                                        bg={false}
                                        className="mt2"
                                        onImgLoad={[Function]}
                                        src="/foo?w=300&fm=png&q=90"
                                        style={Object {}}
                              />
                              <p
                                        className="py2"
                              >
                                        foo
                              </p>
                    </div>
          </Button>
</div>,
      ],
      "className": "w100 p3 mt4 flex content-width mx-auto",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Button
            className="text-white w100"
            color="white-denim"
            disabled={false}
            fullWidth={false}
            hover="shadow"
            label=""
            onClick={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            shadow={false}
            to="foo"
            type="button"
            variant="style-none"
>
            <div
                        className="flex flex-column items-center justify-between p3 w100"
            >
                        <Image
                                    alt=""
                                    bg={false}
                                    className="mt2"
                                    onImgLoad={[Function]}
                                    src="/foo?w=300&fm=png&q=90"
                                    style={Object {}}
                        />
                        <p
                                    className="py2"
                        >
                                    foo
                        </p>
            </div>
</Button>,
          "className": "col-12 md-col-6 m1",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <div
              className="flex flex-column items-center justify-between p3 w100"
>
              <Image
                            alt=""
                            bg={false}
                            className="mt2"
                            onImgLoad={[Function]}
                            src="/foo?w=300&fm=png&q=90"
                            style={Object {}}
              />
              <p
                            className="py2"
              >
                            foo
              </p>
</div>,
            "className": "text-white w100",
            "color": "white-denim",
            "disabled": false,
            "fullWidth": false,
            "hover": "shadow",
            "label": "",
            "onClick": [Function],
            "onMouseEnter": [Function],
            "onMouseLeave": [Function],
            "shadow": false,
            "to": "foo",
            "type": "button",
            "variant": "style-none",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <Image
                  alt=""
                  bg={false}
                  className="mt2"
                  onImgLoad={[Function]}
                  src="/foo?w=300&fm=png&q=90"
                  style={Object {}}
/>,
                <p
                  className="py2"
>
                  foo
</p>,
              ],
              "className": "flex flex-column items-center justify-between p3 w100",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "alt": "",
                  "bg": false,
                  "children": null,
                  "className": "mt2",
                  "onImgLoad": [Function],
                  "src": "/foo?w=300&fm=png&q=90",
                  "style": Object {},
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "foo",
                  "className": "py2",
                },
                "ref": null,
                "rendered": "foo",
                "type": "p",
              },
            ],
            "type": "div",
          },
          "type": [Function],
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Button
            className="text-white w100"
            color="white-denim"
            disabled={false}
            fullWidth={false}
            hover="shadow"
            label=""
            onClick={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            shadow={false}
            to="foo"
            type="button"
            variant="style-none"
>
            <div
                        className="flex flex-column items-center justify-between p3 w100"
            >
                        <Image
                                    alt=""
                                    bg={false}
                                    className="mt2"
                                    onImgLoad={[Function]}
                                    src="/foo?w=300&fm=png&q=90"
                                    style={Object {}}
                        />
                        <p
                                    className="py2"
                        >
                                    foo
                        </p>
            </div>
</Button>,
          "className": "col-12 md-col-6 m1",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <div
              className="flex flex-column items-center justify-between p3 w100"
>
              <Image
                            alt=""
                            bg={false}
                            className="mt2"
                            onImgLoad={[Function]}
                            src="/foo?w=300&fm=png&q=90"
                            style={Object {}}
              />
              <p
                            className="py2"
              >
                            foo
              </p>
</div>,
            "className": "text-white w100",
            "color": "white-denim",
            "disabled": false,
            "fullWidth": false,
            "hover": "shadow",
            "label": "",
            "onClick": [Function],
            "onMouseEnter": [Function],
            "onMouseLeave": [Function],
            "shadow": false,
            "to": "foo",
            "type": "button",
            "variant": "style-none",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <Image
                  alt=""
                  bg={false}
                  className="mt2"
                  onImgLoad={[Function]}
                  src="/foo?w=300&fm=png&q=90"
                  style={Object {}}
/>,
                <p
                  className="py2"
>
                  foo
</p>,
              ],
              "className": "flex flex-column items-center justify-between p3 w100",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "alt": "",
                  "bg": false,
                  "children": null,
                  "className": "mt2",
                  "onImgLoad": [Function],
                  "src": "/foo?w=300&fm=png&q=90",
                  "style": Object {},
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "foo",
                  "className": "py2",
                },
                "ref": null,
                "rendered": "foo",
                "type": "p",
              },
            ],
            "type": "div",
          },
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="col-12 md-col-6 m1"
>
            <Button
                        className="text-white w100"
                        color="white-denim"
                        disabled={false}
                        fullWidth={false}
                        hover="shadow"
                        label=""
                        onClick={[Function]}
                        onMouseEnter={[Function]}
                        onMouseLeave={[Function]}
                        shadow={false}
                        to="foo"
                        type="button"
                        variant="style-none"
            >
                        <div
                                    className="flex flex-column items-center justify-between p3 w100"
                        >
                                    <Image
                                                alt=""
                                                bg={false}
                                                className="mt2"
                                                onImgLoad={[Function]}
                                                src="/foo?w=300&fm=png&q=90"
                                                style={Object {}}
                                    />
                                    <p
                                                className="py2"
                                    >
                                                foo
                                    </p>
                        </div>
            </Button>
</div>,
          <div
            className="col-12 md-col-6 m1"
>
            <Button
                        className="text-white w100"
                        color="white-denim"
                        disabled={false}
                        fullWidth={false}
                        hover="shadow"
                        label=""
                        onClick={[Function]}
                        onMouseEnter={[Function]}
                        onMouseLeave={[Function]}
                        shadow={false}
                        to="foo"
                        type="button"
                        variant="style-none"
            >
                        <div
                                    className="flex flex-column items-center justify-between p3 w100"
                        >
                                    <Image
                                                alt=""
                                                bg={false}
                                                className="mt2"
                                                onImgLoad={[Function]}
                                                src="/foo?w=300&fm=png&q=90"
                                                style={Object {}}
                                    />
                                    <p
                                                className="py2"
                                    >
                                                foo
                                    </p>
                        </div>
            </Button>
</div>,
        ],
        "className": "w100 p3 mt4 flex content-width mx-auto",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Button
              className="text-white w100"
              color="white-denim"
              disabled={false}
              fullWidth={false}
              hover="shadow"
              label=""
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              shadow={false}
              to="foo"
              type="button"
              variant="style-none"
>
              <div
                            className="flex flex-column items-center justify-between p3 w100"
              >
                            <Image
                                          alt=""
                                          bg={false}
                                          className="mt2"
                                          onImgLoad={[Function]}
                                          src="/foo?w=300&fm=png&q=90"
                                          style={Object {}}
                            />
                            <p
                                          className="py2"
                            >
                                          foo
                            </p>
              </div>
</Button>,
            "className": "col-12 md-col-6 m1",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <div
                className="flex flex-column items-center justify-between p3 w100"
>
                <Image
                                alt=""
                                bg={false}
                                className="mt2"
                                onImgLoad={[Function]}
                                src="/foo?w=300&fm=png&q=90"
                                style={Object {}}
                />
                <p
                                className="py2"
                >
                                foo
                </p>
</div>,
              "className": "text-white w100",
              "color": "white-denim",
              "disabled": false,
              "fullWidth": false,
              "hover": "shadow",
              "label": "",
              "onClick": [Function],
              "onMouseEnter": [Function],
              "onMouseLeave": [Function],
              "shadow": false,
              "to": "foo",
              "type": "button",
              "variant": "style-none",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <Image
                    alt=""
                    bg={false}
                    className="mt2"
                    onImgLoad={[Function]}
                    src="/foo?w=300&fm=png&q=90"
                    style={Object {}}
/>,
                  <p
                    className="py2"
>
                    foo
</p>,
                ],
                "className": "flex flex-column items-center justify-between p3 w100",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "alt": "",
                    "bg": false,
                    "children": null,
                    "className": "mt2",
                    "onImgLoad": [Function],
                    "src": "/foo?w=300&fm=png&q=90",
                    "style": Object {},
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "foo",
                    "className": "py2",
                  },
                  "ref": null,
                  "rendered": "foo",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            "type": [Function],
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Button
              className="text-white w100"
              color="white-denim"
              disabled={false}
              fullWidth={false}
              hover="shadow"
              label=""
              onClick={[Function]}
              onMouseEnter={[Function]}
              onMouseLeave={[Function]}
              shadow={false}
              to="foo"
              type="button"
              variant="style-none"
>
              <div
                            className="flex flex-column items-center justify-between p3 w100"
              >
                            <Image
                                          alt=""
                                          bg={false}
                                          className="mt2"
                                          onImgLoad={[Function]}
                                          src="/foo?w=300&fm=png&q=90"
                                          style={Object {}}
                            />
                            <p
                                          className="py2"
                            >
                                          foo
                            </p>
              </div>
</Button>,
            "className": "col-12 md-col-6 m1",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <div
                className="flex flex-column items-center justify-between p3 w100"
>
                <Image
                                alt=""
                                bg={false}
                                className="mt2"
                                onImgLoad={[Function]}
                                src="/foo?w=300&fm=png&q=90"
                                style={Object {}}
                />
                <p
                                className="py2"
                >
                                foo
                </p>
</div>,
              "className": "text-white w100",
              "color": "white-denim",
              "disabled": false,
              "fullWidth": false,
              "hover": "shadow",
              "label": "",
              "onClick": [Function],
              "onMouseEnter": [Function],
              "onMouseLeave": [Function],
              "shadow": false,
              "to": "foo",
              "type": "button",
              "variant": "style-none",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <Image
                    alt=""
                    bg={false}
                    className="mt2"
                    onImgLoad={[Function]}
                    src="/foo?w=300&fm=png&q=90"
                    style={Object {}}
/>,
                  <p
                    className="py2"
>
                    foo
</p>,
                ],
                "className": "flex flex-column items-center justify-between p3 w100",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "alt": "",
                    "bg": false,
                    "children": null,
                    "className": "mt2",
                    "onImgLoad": [Function],
                    "src": "/foo?w=300&fm=png&q=90",
                    "style": Object {},
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "foo",
                    "className": "py2",
                  },
                  "ref": null,
                  "rendered": "foo",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`renders without data 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CardRecirculation
    fields={
        Object {
            "card1Color": "",
            "card1Image": Object {
              "fields": Object {
                "file": Object {
                  "url": "",
                },
                "title": "",
              },
              "sys": Object {
                "id": "",
              },
            },
            "card1Link": "",
            "card1Text": "",
            "card2Color": "",
            "card2Image": Object {
              "fields": Object {
                "file": Object {
                  "url": "",
                },
                "title": "",
              },
              "sys": Object {
                "id": "",
              },
            },
            "card2Link": "",
            "card2Text": "",
            "title": "",
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        null,
        null,
      ],
      "className": "w100 p3 mt4 flex content-width mx-auto",
    },
    "ref": null,
    "rendered": Array [
      null,
      null,
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          null,
          null,
        ],
        "className": "w100 p3 mt4 flex content-width mx-auto",
      },
      "ref": null,
      "rendered": Array [
        null,
        null,
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
